src/nvim/ex_cmds_defs.h

```diff
@@ -63,6 +63,7 @@
 #define EX_MODIFY       0x100000  // forbidden in non-'modifiable' buffer
 #define EX_FLAGS        0x200000  // allow flags after count in argument
 #define EX_KEEPSCRIPT  0x4000000  // keep sctx of where command was invoked
+#define EX_PREVIEW     0x8000000  // allow incremental command preview
```

Comment:
did we jump up here by accident instead of using `0x400000` and `0x8000000` ?

Reply:
seems so, I'll fix it in that case

Reply:
seems like there might be other reasons since changing that makes the tests fail

Reply:
Fixed

Reply:
Well, this is not an accident. Look at Vim's `ex_cmds.h`.

Reply:
reverted the change